docs/guides/observability/event-metrics.adoc

```diff
@@ -21,11 +20,11 @@ If you have multiple instances running in a cluster, you will need to collect th
 
 == Enable event metrics
 
-To start collecting metrics, enable the feature `user-event-metrics`, enable metrics, and enable the metrics for user events.
+To start collecting event metrics, enable metrics, and enable the metrics for user events.
```

Comment:
Nitpick, but I think this would read better without the comma .e.g. "metrics, enable metrics and enable ..."